NFC_EEPROM/source/target/CMakeLists.txt

```diff
@@ -1,7 +1,9 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-mbed_add_cmake_directory_if_labels("TARGET")
+if("M24SR" IN_LIST MBED_TARGET_LABELS)
```

Comment:
Can you also add a condition for the other subdirectory? Just so we do not have to do it again when support for the other target is added.
Thanks